packages/cli/src/utils/platform.ts

```diff
@@ -0,0 +1,26 @@
+export const sanitizePath = (input: string): string => {
+  return input.replace(/[^a-zA-Z0-9 ._\-:\\/@]/g, '');
+};
+
+export const sanitizeLocale = (input: string): string => {
+  return input.replace(/[^a-zA-Z0-9@._-]/g, '');
+};
+
+export function getPlatformArgs(argv: { path?: string; locale?: string; 'project-dir'?: string }) {
```

Comment:
These properties are specific to individual commands, whilst utils are meant to be context-agnostic. Could you move them out to make the getter more general? I’d recommend returning sanitizers instead, so they can be used directly in the command handlers.

Reply:
I made it more agnostic by returning generic `sanitize` function.